spring-web-modules/spring-mvc-file/src/test/java/com/baeldung/file/CustomMultipartFileUnitTest.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.file;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockMultipartFile;
+
+public class CustomMultipartFileUnitTest {
+
+    @Test
+    public void whenProvidingByteArray_thenMultipartFileCreated() throws IOException {
+        byte[] inputArray = "Test String".getBytes();
+        CustomMultipartFile customMultipartFile = new CustomMultipartFile(inputArray);
+        Assertions.assertArrayEquals(inputArray, customMultipartFile.getBytes());
+    }
+
+    @Test
+    public void whenProvidingByteArray_thenMockMultipartFileCreated() throws IOException {
```

Comment:
Since we are using JUnit 5 these methods should not be public, so just remove the modifier.